### DISCUSSION TITLE
How to use a recent version

### DISCUSSION THREAD

**Discussion by Blackilykat** [Discord4J/Discord4J#1258] [Q&A] [answered]
I wanted to write an user-installable app, and noticed that there was [a commit](https://github.com/Discord4J/Discord4J/commit/07713ab) that adds support for that. However the commit is one month old and the latest release is almost a year old. I tried using jetpack (with `implementation 'com.github.Discord4J:Discord4J:c280e74326'`) but since there's multiple projects in one repository it wouldn't work. How can I use a late enough version of discord4j so I can use the user-installable app features?

**Comment by Azn9** [comment, marked as answer]
Hello, for now there is no new version containing this, so you need to use the snapshot version: https://docs.discord4j.com/using-snapshots
